modules/openxr/action_map/openxr_action_map.cpp

```diff
@@ -66,14 +66,14 @@ void OpenXRActionMap::set_action_sets(Array p_action_sets) {
 }
 
 Array OpenXRActionMap::get_action_sets() const {
-	return action_sets;
+	return Array(action_sets);
```

Comment:
So, I guess if `Array(const Array &p_from)` were explicit we would need this, which is sounds like was your process for making this PR.

However, we haven't marked it as explicit yet, and if don't actually do that in the end, then this will just some extra verbosity that may confuse future readers of this code.

Whereas all the changes to the function arguments are good, even if we don't make that constructor explicit, because it follows the style used by Godot everywhere else.

Perhaps this should be left out until we actually make the `Array` copy constructor explicit?

Reply:
In this case, isn't this change better because it clarifies the intent, that a copy is returned? Still, I can remove it from the PR if desired.

Reply:
If we're going to do this everywhere, then I think that's fine. But since we haven't really decided to do that yet, I'd rather not create these handful of unique instances of it